### PR TITLE
Tracing options

### PR DESCRIPTION
**Pull request by klaidliadon**
This PR aims to add some tracing options to the package. See [klaidliadon/trace](https://github.com/klaidliadon/trace) for a practical example of this in action.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module github.com/go-chi/httplog/v2
 
-go 1.21
+go 1.22
```
Comment (pkieltyka) [inline go.mod:3]: Let’s keep it 1.21

**Review by pkieltyka — COMMENTED** [review]
LGTM :) great work and very useful.

I only left a single comment on go.mod

**Comment by VojtechVitek** [comment]
Very nice. This should work well with https://github.com/go-chi/traceid too, right?

**Comment by VojtechVitek** [comment]
Screen Shot 2024-07-15 at 1 02 06 PM

**Pull request merged**